src/TodoListItem.js

```diff
@@ -0,0 +1,9 @@
+import React from "react";
+
+const TodoListItem = (props) => {
+    return (
+        <li>{props.todo.title}</li>
+    )
```

Comment:
we can also destructure pros like

```
const TodoListItem = ({todo}) => {
    return (
        <li>{todo.title}</li>
    )
}
```

this link can be good for taking a look https://medium.com/@lcriswell/destructuring-props-in-react-b1c295005ce0